main.css

```diff
@@ -46,6 +46,12 @@ td{
 	padding:0.2em;
 }
 
+.checklist-icon{
```

Comment:
why is this 18 px, but in the miscChecklistStuff code, it's 16 px? also, we should use em instead of px, so it works with different text sizes. (1 em = width of the "m" character)